recycle-bin.c

```diff
@@ -1,38 +1,34 @@
+#define _WIN32_IE _WIN32_IE_IE70
```

Comment:
This `define` is required for `shlobj.h` to include the `IFileOperation` interface.